roles/epel/tasks/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Check if EPEL is installed
+  stat:
+    path: /etc/yum.repos.d/epel.repo
+  register: epel_installed
+  when: ansible_distribution == "RedHat"
+
+- name: Install EPEL repo
+  yum:
+    name: "https://dl.fedoraproject.org/pub/epel/epel-release-latest-{{ ansible_distribution_major_version }}.noarch.rpm"
+    state: present
+  when: not epel_installed.stat.exists and ansible_distribution == "RedHat"
```

Comment:
`ansible_distribution` could be `CentOS` so it's safer to use `ansible_os_family` here (as you did in other places).